build.gradle

```diff
@@ -24,6 +24,8 @@ dependencies {
 	implementation 'org.ow2.asm:asm-commons:9.0'
 	implementation 'org.ow2.asm:asm-tree:9.0'
 	implementation 'org.ow2.asm:asm-util:9.0'
+
+	testImplementation 'junit:junit:4.13'
```

Comment:
I'd use JUnit 5 instead since it's also used in other Fabric projects like Loader, Stitch, Unpick and Filament.

Reply:
fixed